compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -2474,19 +2509,19 @@ fn show_candidates(
     mode: DiagnosticMode,
     path: Vec<Segment>,
     append: &str,
-) {
+) -> bool {
```

Comment:
Could you add a comment telling what this function returns?

Reply:
Added to the docstring